agent/src/main/java/kanela/agent/util/conf/KanelaConfiguration.java

```diff
@@ -309,7 +311,16 @@ private BootstrapInjectionConfig getBootstrapInjectionConfiguration(Config modul
 
 
     private Try<String> getTypeListPattern(Config config, String path) {
-        return Try.of(() -> List.ofAll(config.getStringList(path)).mkString("|"));
+        return Try.of(() -> {
+          String x = "";
+          try {
+            x = List.ofAll(config.getStringList(path)).mkString("|");
+          } catch (Exception e) {
+            println("SOMETHING WENT VERY VERY WRONG HERE " + e.getMessage());
+            e.printStackTrace();
+          }
+          return x;
```

Comment:
The `try/catch` block is not necessary we are inside of `Try` in the case that you wish add a specific error message please use `Try.onFailure`

Reply:
Oh fuck, this should not have stayed there :joy:, going to fix it right away.